plugin_tests/source_mapnik_test.py

```diff
@@ -117,8 +128,18 @@ def testTileStyleFromGeotiffs(self):
         testImage = os.path.join(
             os.path.dirname(__file__), 'test_files', 'geotiff_style_7_22_51.png')
 
+        testImagePy3 = os.path.join(
+            os.path.dirname(__file__), 'test_files', 'geotiff_style_7_22_51_py3.png')
+
         testImage = PIL.Image.open(testImage)
-        self.assertIsNone(PIL.ImageChops.difference(image, testImage).getbbox())
+        testImagePy3 = PIL.Image.open(testImagePy3)
+
+        diffs = [
+            PIL.ImageChops.difference(image, testImage).getbbox(),
+            PIL.ImageChops.difference(image, testImagePy3).getbbox()
+        ]
+
+        self.assertTrue(None in diffs)
```

Comment:
This can be written as `self.assertIn(None, diffs)`

Reply:
Much better, I will change it.